ui/admin/app/components/form/credential/json/secret-editor/index.hbs

```diff
@@ -0,0 +1,50 @@
+<div class='secret-editor'>
+  {{#if this.canEdit}}
+    <Rose::CodeEditor as |c|>
+      <c.fieldEditor
+        class='json-secret-editor'
+        @onInput={{@onInput}}
```

Comment:
Not related to this PR, but I realized it looks like we don't actually pass `onInput` to the editor in the `code-editor`